### PR TITLE
Cover popular file names per default

### PR DESCRIPTION
**Pull request by thomas-mc-work**

### CODE REVIEW

**Comment by Chris2011** [comment]
I really don't know, whether we need this PR or not, because it is not customizable by the user to handle this. Would you like to explain a bit deeper why you think we should add this to the default?

**Comment by thomas-mc-work** [comment]
These are some of the most common file names on github (the most popular developer platform). This change will improve the out of the box experience and help users to leverage the purpose of this plug-in without further manual interaction.

File: src/main/java/de/markiewb/netbeans/plugins/readmeinprojectview/ReadmeNodeFactory.java [hunk 1]
```diff
@@ -62,7 +62,11 @@
 
     public static final String DEFAULT_FILENAMES = ""
             + ".gitlab-ci.yml\n"
+            + ".gitignore\n"
             + "readme\n"
+            + "README.md\n"
+            + "CHANGELOG.md\n"
+            + "Changelog.md\n"
```
[the remark this answers is not in the document]
Comment (thomas-mc-work) [reply src/main/java/de/markiewb/netbeans/plugins/readmeinprojectview/ReadmeNodeFactory.java]: Ah, all right. I've also added a comment to reflect that in place.

[the remark this answers is not in the document]
Comment (thomas-mc-work) [reply src/main/java/de/markiewb/netbeans/plugins/readmeinprojectview/ReadmeNodeFactory.java]: Next level would be to outsource that list into a properties file.

**Review by Chris2011 — APPROVED** [review]

**Comment by Chris2011** [comment]
Will release a new version soon. I'm new to the process of the new plugin portal so it will take some time :)

**Comment by Chris2011** [comment]
Btw, thx for your PR.

**Pull request merged**